libvuln/jsonblob/jsonblob.go

```diff
@@ -289,12 +435,26 @@ func (s *Store) UpdateEnrichments(ctx context.Context, kind string, fp driver.Fi
 	return ref, nil
 }
 
-// RecordUpdaterStatus is unimplemented
+// RecordUpdaterStatus is unimplemented.
 func (s *Store) RecordUpdaterStatus(ctx context.Context, updaterName string, updateTime time.Time, fingerprint driver.Fingerprint, updaterError error) error {
 	return nil
 }
 
-// RecordUpdaterSetStatus is unimplemented
+// RecordUpdaterSetStatus is unimplemented.
 func (s *Store) RecordUpdaterSetStatus(ctx context.Context, updaterSet string, updateTime time.Time) error {
 	return nil
 }
+
+var bufPool sync.Pool
+
+func getBuf() []byte {
+	const sz = 1 << 20 // 1MiB
+	b, ok := bufPool.Get().([]byte)
+	if !ok {
+		b = make([]byte, sz)
+	}
```

Comment:
why not just implement `bufPool.New`?

Reply:
You need a wrapper function to do the assert anyway, so IMO putting that in the `(sync.Pool).New` just sort of smears the logic across multiple functions.

I'm confident this is not uniform throughout the codebase, but it's what makes sense to me at the moment.